thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/api/dto/EmailFunctionDependency.java

```diff
@@ -0,0 +1,48 @@
+package com.linkedin.thirdeye.api.dto;
```

Comment:
Used for keeping request objects. DTO stands for Data Transfer Object.
There is not entity EmailFunctionDependency anymore, but UI needs an object representation for this, thats where DTOs are used.